app/studio/1.0.x/download-install.md

```diff
@@ -1,50 +1,15 @@
 ---
-title: Download and Install Kong Studio
+title: Download and Install the Kong Studio Bundle
 toc: false
 ---
-## Download Kong Studio
 
-Kong Studio is available to download for Kong Enterprise customers. You must have access to the Kong repository on Bintray to access the Kong Studio download files listed below. If you do not have access, contact Kong Support.
+## Download and Install Kong Studio
```

Comment:
Can you move this header to right before the steps begin (line 11 right now)? I should've compared with the preview, but this pretty much gives the page a double title.